.github/workflows/android.yml

```
@@ -23,4 +23,4 @@ jobs:
     - name: Grant execute permission for gradlew
       run: chmod +x gradlew
     - name: Build with Gradle
-      run: ./gradlew build
+      run: ./gradle build
```

this is working fine before, not the issue here
```suggestion
      run: ./gradlew build
```